src/App.js

```diff
@@ -1,20 +1,26 @@
+import { nanoid } from 'nanoid';
+import { useEffect, useState } from 'react';
+import Confetti from 'react-confetti';
 import './App.css';
 import ColorBlock from './ColorBlock.js';
-import { useState, useEffect } from 'react';
-import { nanoid } from 'nanoid';
-import Confetti from 'react-confetti'
+
+const LETTERS = ['A', 'B', 'C', 'D', 'E', 'F']
```

Comment:
In this case, it's not so relevant in terms of code maintenance since these letters won't change in the future(I think so). But it still, it makes for better readability anyways